ibc/relayer_test.go

```diff
@@ -0,0 +1,35 @@
+package ibc
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestChannelOptsConfigured(t *testing.T) {
+	// Test the default channel opts
+	opts := DefaultChannelOpts()
+	require.True(t, opts.IsFullyConfigured())
```

Comment:
nit:

`IsFullyConfigured` is a mouthful. I would opt for `IsValid`. Programmers are lazy. 😄

Reply:
Additionally, I often see `Validate() error` functions. That way, the error message can tell the user what they are missing.

Reply:
Yeah for sure, `Validate() error` is what I went with for determining the `Order` value is valid but went with `IsFullyConfigured()` to check that all fields were initialized because I noticed another method with the same name and functionality and wanted to keep naming consistent. https://github.com/strangelove-ventures/ibctest/blob/0a2f39ca8fdeac8e36ea2a3a8072aa550e49bc3e/ibc/types.go#L68-L78

I'm open to changing this to `Validate() error` though if that seems better!